mini_pupper_driver/mini_pupper_driver/display_interface.py

```diff
@@ -18,6 +18,7 @@
 # @Author  : Yunlong Feng
 
 import PIL
+import PIL.Image
```

Comment:
from syntax, I don't understand why we need to import this.

since we import PIL already in line 20, it shall be ok as we do in line 42
  ```  
  image = PIL.Image.fromarray(cv2.cvtColor(cv_img, cv2.COLOR_BGR2RGB))
  ```